modules/ROOT/pages/activity-tracking.adoc

```diff
@@ -91,11 +99,17 @@ The details for an outbound transmission include the following information:
 * Name of the transmitted file
 * Number of messages in the transmission
 
+If this transmission is a replay, the details also include the transmission ID of the original (replayed) transmission.
+
+If this transmission was replayed, the details also include the date and time of the most recent replay.
+
 Take any of the following actions to view and download additional information about the outbound transmission:
 
 |===
 |Action |Steps
 
+| View the original transmission if the message is a replay.
+| Click the link in the *Original Transmission Id field*.
```

Comment:
Click the link in the *Original Transmission Id* field.

Reply:
(unless field is part of the UI)

Reply:
OK